.github/workflow_scripts/test_timeseries.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -ex
+
+source $(dirname "$0")/env_setup.sh
+
+setup_build_env
+setup_mxnet_gpu
+export CUDA_VISIBLE_DEVICES=0
+install_core_all_tests
+install_features
+install_tabular_all
```

Comment:
Maybe change in a separate PR, but a bit confuised why we need tabular for timeseries unit tests? @canerturkmen is this needed? Do we need install_features either?

Reply:
They are currently in the Jenkinsfile. So might want inputs from @canerturkmen

Reply:
I don't think this is needed.

Reply:
https://github.com/yinweisu/autogluon/runs/7097339877?check_suite_focus=true Looks like we actually need it.